setup.py

```diff
@@ -4,7 +4,9 @@
        from pip.req import parse_requirements
 import setuptools
 
-VERSION = '0.2.1pre'
+# Loads __version__ using exec as setup.py can't import its own package
+exec(open('learnosity_sdk/_version.py').read())
```

Comment:
Could we `import` this instead?

Reply:
From what I've read, it's not a good idea to import stuff here as package dependencies are not resolved properly. There's some more info here https://milkr.io/kfei/5-common-patterns-to-version-your-Python-package/4

In a later fixup, I've changed this slightly to run the `exec` in a more limited scope. (It also just occurred to me that I should add error handling in case _version.py is missing or doesn't contain the __version__ variable.)